pkg/cloudprovider/aws/launchtemplate.go

```diff
@@ -121,10 +121,16 @@ func (p *LaunchTemplateProvider) createLaunchTemplate(ctx context.Context, clust
 			},
 			TagSpecifications: []*ec2.LaunchTemplateTagSpecificationRequest{{
 				ResourceType: aws.String(ec2.ResourceTypeInstance),
-				Tags: []*ec2.Tag{{
-					Key:   aws.String(fmt.Sprintf(ClusterTagKeyFormat, cluster.Name)),
-					Value: aws.String("owned"),
-				}},
+				Tags: []*ec2.Tag{
+					{
+						Key:   aws.String(fmt.Sprintf(ClusterTagKeyFormat, cluster.Name)),
+						Value: aws.String("owned"),
+					},
+					{
+						Key:   aws.String(fmt.Sprintf(KarpenterTagKeyFormat, cluster.Name)),
```

Comment:
Can you explain this change?

Reply:
Yep, `aws/factory.go` contains:

```go
	// KarpenterTagKeyFormat is set on all Karpenter owned resources.
	KarpenterTagKeyFormat = "karpenter.sh/cluster/%s"
```

So it seems like we are supposed to set this tag on any AWS resources created by Karpenter?